common/plugins/eu.esdihumboldt.hale.common.headless/src/eu/esdihumboldt/hale/common/headless/transform/Transformation.java

```diff
@@ -457,7 +462,27 @@ public void done(IJobChangeEvent event) {
 			storeJob.schedule();
 		}
 		else {
-			// otherwise schedule jobs directly
+			// otherwise feed InstanceProcessors directly from the
+			// InstanceCollection...
+			final InstanceProcessingExtension ext = new InstanceProcessingExtension(
+					serviceProvider);
+			final List<InstanceProcessor> processors = ext.getInstanceProcessors();
+
+			ResourceIterator<Instance> it = sourceToUse.iterator();
+			try {
+				while (it.hasNext()) {
+					Instance instance = it.next();
+
+					ResolvableInstanceReference resolvableRef = new ResolvableInstanceReference(
+							new PseudoInstanceReference(instance), sourceToUse);
```

Comment:
Wouldn't this require rather in-depth knowledge of the performed transformations on the side of the user who uses the CLI?

Maybe it would be sufficient for the moment to simply not do instance processing in cases when OrientDB is not used, since there currently is exactly one `InstanceProcessor` (`SpatialIndexInstanceProcessor`) and its state can only be accessed from Groovy functions, which (if I understand it correctly) will lead to OrientDB usage anyway.